web/src/components/apps/AppGitops.jsx

```diff
@@ -246,9 +262,11 @@ class AppGitops extends Component {
               action={gitops?.action}
               selectedService={selectedService}
               onFinishSetup={this.finishGitOpsSetup}
-              showCancelBtn={!!ownerRepo}
+              showCancelBtn={true}
```

Comment:
what will the cancel button do in case `!ownerRepo` is true and `showGitOpsSettings` is false?

Reply:
Yeah that's a good point, I forgot that could also happen. After changing it back it is behaving as expected.